be/src/http/action/file_cache_action.cpp

```diff
@@ -39,10 +47,13 @@ constexpr static std::string_view SYNC = "sync";
 constexpr static std::string_view PATH = "path";
 constexpr static std::string_view CLEAR = "clear";
 constexpr static std::string_view RESET = "reset";
+constexpr static std::string_view HASH = "hash";
+constexpr static std::string_view HASH_PATH = "hash_path";
```

Comment:
The API name is implicit, maybe list_cache? @gavinchou what do you think?

Reply:
nice idea